plyer/platforms/linux/inappbrowser.py

```diff
@@ -0,0 +1,14 @@
+
+from plyer.facades.inappbrowser import InAppBrowser
+import webbrowser
+
+
+class LinuxAppBrowser():
+
+    def _open_url(self, **kwargs):
+        url = kwargs.get('url', '')
```

Comment:
Using `''` as the default here is bad because `webbrowser` isn't _only_ for opening the internet browser, but it works even as a file browser, therefore `webbrowser.open('')` even if called from console will probably open Nautilus or similar thing. The same thing works for Windows, where it opens `explorer.exe` in the location you call it from e.g. `%cd%` or `pwd` (maybe?).

It most likely works even for OSX in the way that such call opens the default browser (like it does for the rest).

Edit: and does the same thing even for Android, so if iOS behaves in the same way, we could probably skip the whole pyobjus thing and replace it with `webbrowser.open('http(s):// ...')`